.rubocop_styleguide.yml

```diff
@@ -128,3 +128,9 @@ Metrics/PerceivedComplexity:
 
 Naming/PredicateName:
   Enabled: false
+
+Style/InverseMethods:
+  Enabled: true
+  InverseMethods:
+    :include?: ~
```

Comment:
I like the `Style/InverseMethods` cop rule, it's a good one to add 👍 

However I'm curious about this non-default rule, can you explain it?
> `:include?: ~`

This isn't very intuitive to me. My reading of [the docs](https://docs.rubocop.org/rubocop/cops_style.html#styleinversemethods) suggests that this would convert:

```
!myarray.include?(thing)
```
To something like:
```
myarray ~ thing
```

I'm not sure if that's right, but in any case I think using `! include?` is easier to read and would suggest we stick to the default and remove this extra rule.

Reply:
It actually isn't a new rule. I'm just overriding the existing rule.

Anyway, `exclude?` is not a part of the standard Ruby library. That's the reason I was getting all the initial errors. It needs to be added via the [Activesupport gem](https://rubygems.org/gems/activesupport) which is why I disabled it. If we are okay with adding another gem, then yes, I think it should be left in and we can change the inverse `include?`s to `exclude?`

I personally find it easier to read/understand not included than excluded, but that is just me.

Reply:
Hmm, come to think of it, it does seem to flow better when I say it out loud.. let's see what other @openfoodfoundation/core-devs think:

Normally we avoid the `!` operator when there's an equivalent opposite method available (eg change `!something.present?` to `something.blank?`). 
Should we keep the rule for `!include?` -> `exclude?`. Which of these do you prefer?

```ruby
      collection_actions.exclude? action
      !collection_actions.include? action
```

Reply:
`Activesupport` isn't in the [Gemfile](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/Gemfile), so it appears it hasn't been added to this project unless I'm missing something. Were you thinking of `Activerecord`?

Reply:
`Activesupport` is a rails dependency, so it's added as part for Rails. You can see in the Gemfile.lock :: https://github.com/openfoodfoundation/openfoodnetwork/blob/31ffeab4cd8b3657974d0df7d08510bda19cdb2c/Gemfile.lock#L511-L524

Reply:
hmm, I wonder why I'm getting `undefined method` errors for `exclude?` then. I'll have to dig into this.